### DISCUSSION TITLE
q-table problem

### DISCUSSION THREAD

**Discussion by Allihopp** [quasarframework/quasar#9498] [CLI - PWA mode]
Hi everyone.
Have a problem with placing a table, somehow when page width becomes smaller (kinda tablet/mobile view) the table doesn't add x-scroll, instead it just moves far on the left. Why?
![image](https://user-images.githubusercontent.com/29628160/120227769-7de1f800-c252-11eb-9e58-603bb77d8346.png)
Layout is pure default. Only edited Index.vue to add the table sample from docs.

**Comment by ontwikkelfabriek** [comment]
@Allihopp 

> table doesn't add x-scroll,

It does not? Then what is that nested scrollbar I see?
